backend/codingchallenge/api/models/application.py

```diff
@@ -1,29 +1,13 @@
 from django.db import models
-from django.contrib.auth.models import User
-
-
-# The Default Values can be looked up at: https://docs.djangoproject.com/en/4.1/ref/models/fields/
-
-class Status(models.TextChoices):
-    NOTHING = 0,
-    ACTIVE = 1,
-    PASSIVE = 2
-
-
-class Challenge(models.Model):
-    challengeHeading = models.CharField(max_length=100)
-    challengeText = models.CharField(max_length=5000)
-
-    def __str__(self):
-        return self.challengeId
-
+from .status import Status
 
 class Application(models.Model):
```

Comment:
What does the "Application" mean?
If it's meant to be "Applicant" please rename it (same goes for the file name)

Reply:
As described in Wiki, there is no applicantId in the System of amplimimd. We made a mistake, because we named it `applicant` prior. But a applicant can have multiple applications and we got a requirement, to look at each application on its own.

Reply:
Ok mb